source/in_the_box.rst

```diff
@@ -114,6 +114,12 @@ below.
    :start-after: embed-user-accounts-start
    :end-before: embed-user-accounts-end
 
+Connecting a Monitor
+~~~~~~~~~~~~~~~~~~~
+If you are unable to access the robot through ssh due to your network settings you will need to connect an hdmi monitor, usb keyboard and usb mouse to the side panel ports. Then use them to select your network as you would a normal ubuntu computer. Once it is set it should remain connected through reboots.
+
+.. warning::
+    The monitor cables that conform to hdmi specifications will fit the robot's monitor port. But some cheaper cables have overmolding that is too large given the spec and may not seat properly. Please ensure the hdmi cable you are using is the correct size.
```

Comment:
The leading 'the' can be dropped.

Also probably should use capitalization for proper nouns/acronyms for USB, HDMI, Ubuntu, maybe SSH.